packages/components/src/modal/docs/Modal.stories.mdx

```diff
@@ -49,7 +50,7 @@ A modal must have an heading and a content.
 
 ### Image
 
-A modal can have a side banner [image](?path=/story/chromatic-image--default):
+A modal can have a side banner [image](?path=/story/chromatic-image--default). Make sure the image has no essential information as it could be cropped in mobile view. Images should not prevent a user from seeing the close button, be conscious of this.
```

Comment:
Removing it would in most real world use cases render the modal bland, as we use if for buying process, I'd rather have a different image.

> Are you planning on updating the example image before releasing since the current one prevent the user from seeing the close button?

Yes #921 (in progress) would make it easy to do, at worst I could use the useResponsiveValue until 921 is merged. @Matthieu-Ortalda is going to produce a lighter image too.